_episodes/07-dimuonspectrum.md

```diff
@@ -343,14 +345,13 @@ You could use the `np.logspace()` function for the binning. It helps in returnin
 > ## Solution
 >```python
 >logbins = np.logspace(0, 2.5, 200)
->
->plt.hist(pm, bins=logbins, histtype="step")
->
->plt.xlabel("mass (GeV/$c^2$)")
->plt.ylabel("Events")
->plt.xscale("log")
->plt.title("Mass of dimuons per event")
->plt.autoscale()
+>fig, ax = plt.subplots()
+>ax.hist(pm, bins=logbins, histtype="step")
+>ax.xlabel("mass (GeV/$c^2$)")
```

Comment:
Isn't it `set_xlabel`? Probably best to quickly run this snippet locally to check that it still works :) 
(also not sure about `autoscale`)

Reply:
I tested autoscale it worked. 
yeah its set_xlabel (my mistake)

Reply:
DONE